src/big/big_decimal.cr

```diff
@@ -732,7 +756,7 @@ struct BigDecimal < Number
   end
 
   private def power_ten_to(x : Int) : Int
-    TEN_I ** x
+    x == 1 ? TEN_I : TEN_I ** x
```

Comment:
Is this the right place for this optimization? In `BigInt#**` it would be generically useful.